activerecord/test/cases/migration/foreign_key_test.rb

```diff
@@ -19,6 +19,52 @@ def test_foreign_keys
           assert_equal "fk_name", fk.name unless current_adapter?(:SQLite3Adapter)
         end
       end
+
+      if current_adapter?(:SQLite3Adapter)
```

Comment:
The condition is unnecessary since the test case should pass for all adapters which support foreign keys.

Reply:
It got failures with all bundled adapters https://travis-ci.org/rails/rails/builds/381658874 then reverted change then now it is tested with `sqlite3` adapters only.

Reply:
The reason which got failures is that dropping tables defined in https://github.com/rails/rails/blob/master/activerecord/test/schema/schema.rb after loaded.

Reply:
ah, you are right.

Reply:
Now using `Rocket` and `Astronaut` models not to conflict with other test files.